Detectors/Base/include/DetectorsBase/Detector.h

```diff
@@ -154,6 +154,13 @@ class Detector : public FairDetector
     // (typically the case for geometry related stuff, etc)
     virtual void InitializeO2Detector() = 0;
 
+    // the FairModule/Detector virtual DefineSensitiveVolume() function
+    // is now done in InitializeO2Detector()
+    void DefineSensitiveVolumes() final
+    {
+      InitializeO2Detector();
+    }
```

Comment:
This will likely lead to a double call of InitializeO2Detector since this function is also called from another place.

It might be better to actually implement this interface in all detectors and pull it out from Init. Now it is very confusing. TPC implements "defineSensitiveVolumes" which is quite (too) close to the required interface "DefineSensitiveVolumes".

Can you also shed some light into why this is important for MT mode but not for scalar mode?

Reply:
I will double check this. Would you remember in which PR you introduced the InitializeO2Detector() function? As it seems that this change triggered the warning which we observe now.